### PR TITLE
Moving configurations from boxes to machines

### PR DESCRIPTION
**Pull request by VimukthiPerera**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? no
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: boxes/identity-server-analytics/provisioner/provisioner.sh [hunk 1]
```diff
@@ -59,12 +65,20 @@ if test -d ${JAVA_HOME}; then
 fi
 echo "Successfully set up Java"
 
-# unpack the WSO2 product pack to the working directory
-echo "Setting up the ${WSO2_SERVER}-${WSO2_SERVER_VERSION} server..."
+# set up WUM
+echo "Setting up WUM..."
+if test ! -d ${WUM_HOME}; then mkdir ${WUM_HOME}; fi
+if test -d ${WUM_HOME}; then
+  tar -xzf ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} -C ${WUM_HOME} --strip-components=1
```
Comment (chirangaalwis) [inline boxes/identity-server-analytics/provisioner/provisioner.sh]: @VimukthiPerera why do we need `z` argument for extracting the WSO2 WUM distribution in this case? I believe `tar -xf ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} -C ${WUM_HOME} --strip-components=1` would suffice.

File: boxes/identity-server/provisioner/provisioner.sh [hunk 2]
```diff
@@ -60,12 +66,20 @@ if test -d ${JAVA_HOME}; then
 fi
 echo "Successfully set up Java"
 
-# unpack the WSO2 product pack to the working directory
-echo "Setting up the ${WSO2_SERVER}-${WSO2_SERVER_VERSION} server..."
+# set up WUM
+echo "Setting up WUM..."
+if test ! -d ${WUM_HOME}; then mkdir ${WUM_HOME}; fi
+if test -d ${WUM_HOME}; then
+  tar -xzf ${SOFTWARE_DISTRIBUTIONS}/${WUM_ARCHIVE} -C ${WUM_HOME} --strip-components=1
```
Comment (chirangaalwis) [inline boxes/identity-server/provisioner/provisioner.sh]: Please refer the above review regarding the usage of `z` argument in `tar` command.

File: boxes/identity-server-analytics/provisioner/provisioner.sh (hunk 1 above)
Comment (VimukthiPerera) [reply boxes/identity-server-analytics/provisioner/provisioner.sh]: @chirangaalwis yes. After version  1.15 "z" argument is not needed. But in the WUM installation guide it was mentioned as above https://wso2.com/wum/download . I'll change this to "-xf"

File: boxes/identity-server/provisioner/provisioner.sh (hunk 2 above)
Comment (VimukthiPerera) [reply boxes/identity-server/provisioner/provisioner.sh]: fixed

**Pull request merged**